### PR TITLE
project management practices

### PR DESCRIPTION
**Pull request by pkalliok**
Bring all kinds of project management related stuff from our wiki to Github.  Also, fix link syntax for links.

### CODE REVIEW

File: project-management.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+* Reviews: one hour to show the sprint increment to stakeholders and get feedback
+
+  * Reviews are split by the service worked on
+  * This is the place to make a final check on how the sprint went
+  * try to get comments and suggestions for further work
+* Retrospectives: one hour to improve on ways of working
+  * check on earlier retrospective items and how they are progressing
+  * tell things that suck in our way of working
+  * pick the most painful ones (e.g. by vote)
+  * decide what is to be done about them, when and by whom
```
Comment (tituomin) [inline project-management.md]: @pkalliok These are a bit KUVA-specific. But still the DoR and DoD parts seem like a good base for city-wide common definitions.

File: project-management.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  * During sprint: see that the sprint is proceeding at least somewhat according to the plan; raise the issue and investigate reasons
+    *   Check that reviews are being done quickly (check their WIP) - remind developers to push stuff into review, preferably f2f with the reviewer
+    *   Check that PO is accepting reviewed work (check “acceptance” WIP) - remind developers to push issues into acceptance
+  * Help team members keep each other accountable when reviewing stories and pull requests: Do they fulfil requirements in DoR and DoD
+  * Identify, keep track of and resolve impediments (also in form of work blocked because of dependencies).
+
+In general, see:
+
+  * [scrum master responsibilities](https://scrummasterchecklist.org/pdf/ScrumMaster_Checklist_12_unbranded.pdf)
+  * [scrum checklist](https://www.crisp.se/wp-content/uploads/2012/05/Scrum-checklist.pdf)
```
Comment (tituomin) [inline project-management.md]: Somewhat KUVA specific also, but maybe leave this here for those projects that do have Scrum Masters.

**Review by tituomin — CHANGES_REQUESTED** [review]

File: project-management.md (hunk 1 above)
Comment (pkalliok) [reply project-management.md]: Handled by making a Scrum-specific title for this.

File: project-management.md (hunk 2 above)
Comment (pkalliok) [reply project-management.md]: Handled by making a scrum-specific title for this.

**Review by tituomin — APPROVED** [review]

**Pull request merged**